coeus-impl/src/main/java/org/kuali/kra/irb/ProtocolDocument.java

```diff
@@ -163,6 +171,13 @@ protected void mergeAmendment(String protocolStatusCode, String type) {
         }
         try {
             getDocumentService().saveDocument(newProtocolDocument);
+            //manually set the principleID of the last actionrequest to BLANK, so this routes
+            List requests = getActionRequestService().findAllActionRequestsByDocumentId(newProtocolDocument.getDocumentHeader().getDocumentNumber());
+            if (requests != null && requests.size() > 0) {
+            	ActionRequestValue lastRequest = (ActionRequestValue)requests.get(requests.size() -1);
+            	lastRequest.setPrincipalId("");
+            	getDataObjectService().save(lastRequest);
+            }
             // blanket approve to make the new protocol document 'final'
```

Comment:
The last action request, is the action the approved the merge of the protocol with the amendment.  If there is a problem with that, it would throw an exception before we get here.  By the time we get to this code, we just need to route the document to finalize it.

As per StandardWorkflowEngine.checkDefaultApprovalPolicy,  for the route to be approved, the document's initiator workflow ID, and the action request's principle ID must be different.  The code I have just resets the principle ID, so it will pass this test.

I think the ActionRequestValue in question here is being set by earlier processing in Rice.